recipes/tree-sitter-indent

```diff
@@ -0,0 +1,3 @@
+(tree-sitter-indent :fetcher git
+                    :url "https://codeberg.org/FelipeLema/tree-sitter-indent.el.git"
+                    :branch "main")
```

Comment:
Just quickly noting - if you're specifying `:branch "main"` for compatibility with straight.el, and `main` is your default branch, I believe straight.el has fixed this.  Generally we discourage specifying branch except in unusual cases.

Reply:
oh, I see... I'll remove this property, then.